draft-ietf-privacypass-protocol.md

```diff
@@ -12,984 +12,423 @@ stand_alone: yes
-ignores the first message and uses an empty value for the commitment
-parameter to `Generate`.
+## Issuer Configuration {#public-issuer-configuration}
 
-## PP(OPRF2)
+Issuers are configured with Private and Public Key pairs, each denoted skI and
+pkI, respectively, used to produce tokens. Each key pair MUST be generated as
+as a valid 4096-bit RSA private key according to [TODO]. The key identifier
+for a keypair (skI, pkI), denoted `key_id`, is computed as SHA256(encoded_key),
+where encoded_key is a DER-encoded SubjectPublicKeyInfo object carrying pkI.
 
-- OPRF2 = OPRF(decaf448, SHA-512)
-- ID = 0x0001
-- COMMIT = false
-- Maximum security provided: 224 bits
+# Security considerations
 
-## PP(OPRF4)
+This document outlines how to instantiate the Issuance protocol
+based on the VOPRF defined in {{I-D.irtf-cfrg-voprf}}. All security
+considerations described in the VOPRF document also apply in the Privacy
+Pass use-case. Considerations related to broader privacy and security
+concerns in a multi-Client and multi-Issuer setting are deferred to the
+Architecture document {{I-D.ietf-privacypass-architecture}}.
 
-- OPRF4 = OPRF(P-384, SHA-512)
-- ID = 0x0002
-- COMMIT = false
-- Maximum security provided: 192 bits
+# IANA considerations
 
-## PP(OPRF5)
+This document updates the "Token Type" Registry with the following values.
 
-- OPRF5 = OPRF(P-521, SHA-512)
-- ID = 0x0003
-- COMMIT = false
-- Maximum security provided: 256 bits
-
-# Extensions framework policy {#extensions}
-
-The intention with providing the Privacy Pass API in {{pp-api}} is to
-allow new instantiations of the Privacy Pass protocol. These
-instantiations may provide either modified VOPRF constructions, or
-simply implement the API in a completely different way.
-
-Extensions to this initial draft SHOULD be specified as separate
-documents taking one of two possible routes:
-
-- Produce new VOPRF-like primitives that use the same public API
-  provided in {{I-D.irtf-cfrg-voprf}} to implement the Privacy Pass API,
-  but with different internal operations.
-- Implement the Privacy Pass API in a different way to the proposed
-  implementation in {{voprf-protocol}}.
-
-If an extension requires changing the generic protocol description as
-described in {{overview}}, then the change may have to result in changes
-to the draft specification here also.
-
-Each new extension that modifies the internals of the protocol in either
-of the two ways MUST re-justify that the extended protocol still
-satisfies the security requirements in {{sec-reqs}}. Protocol extensions
-MAY put forward new security guarantees if they are applicable.
-
-The extensions MUST also conform with the extension framework policy as
-set out in the architectural framework document. For example, this may
-concern any potential impact on client anonymity that the extension may
-introduce.
+| Value  | Name                   | Public | Public Metadata | Private Metadata | Nk  | Reference        |
```

Comment:
Public -> Publicly Verifiable since we have a  Public Metadata column now?

Reply:
Yep, good suggestion.